src/pages/loading-waiting-states/progress-bar/index.tsx

```diff
@@ -0,0 +1,234 @@
+import React, { useState, useCallback, useEffect } from 'react';
+import {
+    AppBar,
+    Button,
+    Card,
+    Toolbar,
+    Typography,
+    Hidden,
+    IconButton,
+    Radio,
+    RadioGroup,
+    FormControlLabel,
+    FormControl,
+    List,
+} from '@material-ui/core';
+import { makeStyles, Theme } from '@material-ui/core/styles';
+import MenuIcon from '@material-ui/icons/Menu';
+import { Folder, VerticalAlignTop, Description } from '@material-ui/icons';
+import LinearProgress, { LinearProgressProps } from '@material-ui/core/LinearProgress';
+import { useDispatch } from 'react-redux';
+import { TOGGLE_DRAWER } from '../../../redux/actions';
+import { InfoListItem } from '@pxblue/react-components';
+import * as Colors from '@pxblue/colors';
+type FolderItem = {
+    id: string;
+    name: string;
+    progress: number;
+    status: string;
+};
+const foldersList = [
+    { label: 'The Best Dev Team', value: '1' },
```

Comment:
Why do we want a `value` field here?

Reply:
I assume it's for the Radio Group picker...each item has a label and a value. We could use the same thing for both, but it looks like clicking the radio buttons doesn't do anything right now anyway.